docs/families/blackcat.md

```diff
@@ -15,13 +15,13 @@ parent: Ransomware Families
 |**Artifacts** | | |
 |Extensions | 7 characters in length, specified in config file | [^5] |
 |Ransomware Notes | RECOVER-[EXTENSION]-FILES.txt | [^5] |
-|Services It Disables | TBD | |
+|Services It Disables | IIS, Antivirus, backup service, recovery tool in Windows boot menu and clear Windows event logs |[^8] |
 |Other Observables | Also named ALPHV and Noberus | [^6]|
 |**Automation** | | |
-|Initial Access | No |  |
+|Initial Access | Yes (exploiting Exchange servers, using remote access tools or getting compromised credentials within dark markets) |[^7] [^8]  |
```

Comment:
Same with Exfil comment below. I think the threat actor needs to obtain initial access w/out using the BlackCat EXE. Therefore, those techniques are not listed on this page (which is specific to the EXE).

Thanks.